include/class.forms.php

```diff
@@ -3912,7 +3912,7 @@ function uploadAttachment(&$file) {
 
     static function isValidFile($file) {
```

Comment:
I think making it optional by default is ideal then we can add an option on Attachments Settings to enable strict mime type check. 

```php
 static function isValidFile($file, $strict = false) { 
        // Check Mime Type
      if ($strict 
           && !empty($file['type']) 
           && strcasecmp(FileObject::mime_type($file['tmp_name']), $file['type']) !== 0)

```

Reply:
Done - waiting for github to update 🙄